src/components/BannerBlock.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from "styled-components";
+
+interface BannerBlockProps {
+    content: {
+        text: string;
+        button: {
+            content: string;
+            link: string;
+        }
+    }
+}
+
+const BannerBlock = (props: BannerBlockProps) => (
+    <ApplicationDiv>
+      <FlexContainer>
+        <Text>{props.content.text}</Text>
+      </FlexContainer>
+      <FlexContainer>
+        <Button onClick={() => {}}>{props.content.button.content}</Button>
+      </FlexContainer>
+    </ApplicationDiv>
+);
+
+const ApplicationDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
```

Comment:
there is too much space on the left and right of the banner and the responsiveness isn't the best (text is moving outside of the banner when resizing window)
here is what I would do to fix it:
1. change ApplicationDiv's justify-content to space-evenly
2. remove margins from Text and Button
3. remove width from FlexContainer
4. add display: flex to FlexContainer
5. add align-items: center to FlexContainer
6. change Text's text-align to center